js/apps/admin-ui/src/clients/authorization/ResourcesPolicySelect.tsx

```diff
@@ -108,57 +128,141 @@ export const ResourcesPolicySelect = ({
     [search],
   );
 
+  const [toggleUnsavedChangesDialog, UnsavedChangesConfirm] = useConfirmDialog({
+    titleKey: t("unsavedChangesTitle"),
+    messageKey: t("unsavedChangesConfirm"),
+    continueButtonLabel: t("common:continue"),
+    continueButtonVariant: ButtonVariant.danger,
+    onConfirm: () => {
+      navigate(
+        toPolicyDetails({
+          realm: realm,
+          id: clientId,
+          policyId: clickedPolicy?.id!,
+          policyType: clickedPolicy?.type!,
+        })
+      );
+    },
+  });
+
   const toSelectOptions = () =>
     items.map((p) => (
       <SelectOption key={p.id} value={p.id}>
         {p.name}
       </SelectOption>
     ));
 
+  const toChipGroupItems = (
+    field: ControllerRenderProps<PolicyRepresentation, Type>
+  ) => {
+    return (
+      <ChipGroup>
+        {field.value?.map((permissionId) => {
+          const policy = items.find(
+            (permission) => permission.id === permissionId
+          );
+
+          if (!policy) {
+            return null;
+          }
+
+          const to = toPolicyDetails({
```

Comment:
This link is also used higher up in the unsaved changes dialog. so perhaps it can be re-used there as well?